tools/ld-analyse/ld-analyse.pro

```diff
@@ -159,6 +160,7 @@ unix:!macx {
 win32: {
     LIBS += qwt.lib
 }
+win32:INCLUDEPATH += "D:\prog\vcpkg\installed\x64-windows\include\qwt"
```

Comment:
Is that addition intentional? I assume that's a path on your machine...

Reply:
Ah yeah that  just me forgetting to remove it